src/server/modes/opensource/routes.ts

```diff
@@ -42,6 +50,46 @@ export function getRoutes({
     return routes;
 }
 
+function getZitadelRoutes({
```

Comment:
It will be better to extract this to the separate file

Reply:
yea, extracted either to Zitadel related stuff